nvbench/device_info.cuh

```diff
@@ -175,6 +183,30 @@ struct device_info
   /// @return True if ECC is enabled on this device.
   [[nodiscard]] bool get_ecc_state() const { return m_prop.ECCEnabled; }
 
+  /// @return True if CUPTI supports this device.
+  [[nodiscard]] bool is_CUPTI_supported() const
```

Comment:
Some API capitalizes `CUPTI`, some just uses lowercase `cupti`. Let's just stick with lowercase throughout the API, since other abbreviations are handled that way.